src/main/java/io/github/townyadvanced/townyresources/commands/TownyResourcesAdminCommand.java

```diff
@@ -51,10 +53,13 @@ private void parseAdminCommand(CommandSender sender, String[] args) {
 			switch (args[0]) {
 				case "reload":
 					parseReloadCommand(sender);
-				break;
+					break;
 				case "reroll_all_resources":
 					parseReRollCommand(sender);
-				break;
+					break;
+				case "bypass":
+					bypassExtractionLimitCommand(sender);
+                    break;
```

Comment:
Last thing: this line is using spaces instead of tabs, would you mind switching it to use tabs?

Reply:
Done